docs/DeveloperGuide.md

```diff
@@ -487,6 +487,19 @@ testers are expected to do more *exploratory* testing.
     1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
        Expected: Similar to previous.
 
+3. Deleting persons while all persons are being shown
+
+    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+
+    1. Test case: `delete 1 2`<br>
+       Expected: First and second contact are deleted from the list. Names of the deleted contacts are shown in the status message. Timestamp in the status bar is updated.
+
+    1. Test case: `delete x y`<br> (where x,y are greater than the number of listed persons)
+       Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+
+    1. Test case: `delete 0 1`<br> 
```

Comment:
My testing seems to show that the command `delete 0 1` returns the "Invalid command format" exception, and doesn't delete the contact at index 1. I'm suspecting that this is due to the ParserUtil's `parseIndex` throwing a ParseError in the presence of a 0 index given to the delete command. 

IMO, this would also be a reasonable outcome to this user input, so maybe we can leave this be for MVP if we're tight on time!